kale/loaddata/numeric_acccess.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import pandas as pd
+
+def read_csv_numeric(csv_path:str, index_col:list) -> np.array:
+    """Read numeric data from comma separated text.
```

Comment:
`tabular` should be a more precise word for this module than `numeric`, i.e., `numeric_access` to `tabular_access`, and similarly for the rest

Reply:
@sz144 What do you think about `tabular` vs `numeric`?

Reply:
Tabular is more general, while numeric is more precise. I prefer `numeric` if it will not support other data type in future.

Reply:
We do need to support EHR data via `Tabular`, e.g. what Mwiza did on our ABIDE data and what Mohammod is working on our cardiac data.